examples/express/with-emailpassword/backend/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "supertokens-node",
+    "version": "0.0.1",
+    "private": true,
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "start": "npx ts-node-dev --project ./tsconfig.json ./index.ts"
+    },
+    "dependencies": {
+        "cors": "^2.8.5",
+        "express": "^4.18.1",
+        "helmet": "^5.1.0",
+        "morgan": "^1.10.0",
+        "npm-run-all": "^4.1.5",
+        "supertokens-node": "latest",
```

Comment:
The supertokens-node in the example app should import from this repo itself. Can you make it so that this dependency points to the code in this repo? Or else you can even remove this dependency and in the app, import like "../../../" or something.